test/node/download-webseed-torrent.js

```diff
@@ -9,8 +9,12 @@ var serveStatic = require('serve-static')
 var test = require('tape')
 var WebTorrent = require('../../')
 
+// it should be fast to download a small torrent over local HTTP
+var WEB_SEED_TIMEOUT_MS = 500
+
 test('Download using webseed (via .torrent file)', function (t) {
   t.plan(6)
+  t.timeoutAfter(WEB_SEED_TIMEOUT_MS)
```

Comment:
Won't this just add flakiness to the test suite? We don't do this in any of the other tests. What if it takes 600ms to complete on Travis CI servers?